### PR TITLE
Cleanup CSS code

### LINKED ISSUE #458: CSS Code Cleanup
Go through the CSS files and make some updates, check comments, re-order some code. 

Noticing in base.css there is a comment about styles being there that should be moved. Want to make sure there aren't other bits of CSS that are in the wrong files.

### PR DESCRIPTION
**Pull request by ststimac**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

## Short description of the change(s)

Fix #458 

Cleans up CSS Code, removing the following unnecessary styles and components: 

- [x] Remove workarounds for .module--primary typography overrides 
- [x] Remove styling that implemented a decorative `<hr>` type bar above titles (which caused the need for workarounds)
- [x] Remove module code for "placement"
- [x] Remove module code for "tabs"

### CODE REVIEW

**Comment by molant** [comment]
@ststimac looks like `placement.css` is used in some templates and that's why the build is failing. Can you please take a look?

**Comment by ststimac** [comment]
@molant yup, on it.

**Comment by ststimac** [comment]
I didn't realize it was being used on the Docs > Hints page. Re-added.

File: src/webhint-theme/source/components/placement/placement.css [hunk 1]
```diff
@@ -47,4 +47,4 @@
 	bottom: 0;
 	left: 0;
 	content: "";
-}
+}
```
Comment (sarvaje) [inline src/webhint-theme/source/components/placement/placement.css]: Nit: Add new line to the end of the file.

**Comment by ststimac** [comment]
Updated w/ @sarvaje's edit.

**Pull request merged**